src/lib/import-export/data-export.ts

```
@@ -0,0 +1,1001 @@
+      case 'RISKS':
+        totalRecords = await this.prisma.risk.count({ where: { organizationId } });
+        break;
+      case 'CONTROLS':
+        totalRecords = await this.prisma.control.count({ where: { organizationId } });
+        break;
+      case 'ALL_DATA':
+        const [riskCount, controlCount, userCount] = await Promise.all([
+          this.prisma.risk.count({ where: { organizationId } }),
+          this.prisma.control.count({ where: { organizationId } }),
+          this.prisma.user.count({ where: { organizationId } }),
+        ]);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)